spec/factories/efile_submissions.rb

```diff
@@ -48,7 +48,7 @@
 
     trait :for_state do
       tax_return {}
-      data_source { create :state_file_ny_intake }
+      data_source { create :state_file_az_intake }
```

Comment:
hoping this doesnt catastrophically mess things up

Reply:
it didnt! had to do a small follow up commit to fix https://github.com/codeforamerica/vita-min/pull/5445/commits/42ced40b3457f7205df7b0357f79911e31b53f3f